test/csharp/testing-matchers/test.ts

```diff
@@ -0,0 +1,21 @@
+import { TestDriver } from "../../test-helper";
+
+describe("csharp testing assertions", () => {
+  let driver: TestDriver;
+
+  beforeAll(async () => {
+    driver = new TestDriver(__dirname);
+    await driver.setupCsharpProject();
+    await driver.copyFiles("Test.cs");
+    await driver.copyFiles("update-csproj.sh");
+    await driver.exec("dotnet add package HashiCorp.Cdktf.Providers.Aws");
```

Comment:
I think we should use manual provider generation here instead of prebuilt providers